templates/label-portrait.php

```diff
@@ -30,7 +30,7 @@
     ?>
   </style>
 </head>
-<body>
+<body class="A<?= esc_attr($_GET['format'][1]) ?>-body-wrapper">
```

Comment:
@chrisbaltazar should we better use the built-in `sanitize_text_field ` function from WP?